ci/hardware_tests/hardware.cfg

```diff
@@ -0,0 +1,4 @@
+[usb]
```

Comment:
why is the section called `usb`? wouldn't something like `t1` be more appropriate?

Reply:
Done